composer.json

```diff
@@ -20,28 +20,28 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "illuminate/support": "5.1.* || 5.2.*",
-        "illuminate/http": "5.1.* || 5.2.*",
-        "illuminate/auth": "5.1.* || 5.2.*",
-        "illuminate/contracts": "5.1.* || 5.2.*",
-        "namshi/jose": "6.0.*",
-        "nesbot/carbon": "^1.0"
+        "illuminate/auth": "~5.1.0|~5.2.0",
```

Comment:
This is dangerous since we don't know if it will break in future versions of Laravel.